web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Installing Smart Proxy",
+		href: "/Installing_Proxy_on_Red_Hat/index-foreman-deb.html"
```

Comment:
This page also looks empty.

Reply:
Yeah, I made a patch that does render an empty page on guides which are not correct.

How bad is Installing Smart Proxy for both EL and Debian? I think we might hide this one for now.

Reply:
I am hiding this for now.

Reply:
It needs a bunch of work so I'm good with hiding for now.